### DISCUSSION TITLE
DAI 2024-2025 - Practical work 3 - Urs Behrmann, Rodrigo Lopes dos santos

### DISCUSSION THREAD

**Discussion by UBehrmann** [heig-vd-dai-course/heig-vd-dai-course#552] [Show and tell]
Une API pour interagir avec notre base de données PostgreSQL sera développée dans le cadre du projet BDR.

Notre base de données est utilisée pour gérer les utilisateurs ainsi que leurs accès à diverses données de capteurs.

Nous allons concevoir une interface HTML basique qui utilisera cette API pour afficher et modifier les données.

Les fonctionnalités prévues incluent :

- Connexion au site,
- Création d'utilisateurs,
- Modification des informations de son compte,
- Ajout et suppression de personnes dans un groupe d'utilisateurs,
- Ajout et suppression d'appareils associés à un groupe d'utilisateurs,
- Visualisation des données collectées par un ou plusieurs appareils.

**Comment by GeraudSilvestri** [comment]
Très bien

**Comment by UBehrmann** [comment]
Repo : https://github.com/UBehrmann/DAI_PW3

Dernier commit : https://github.com/UBehrmann/DAI_PW3/commit/9aa090bc086a8fefb3736b514c00e1d7b339783f
